src/coreclr/scripts/superpmi_setup.py

```diff
@@ -108,12 +152,15 @@
     "mscordbi.dll",
     "mscorrc.dll",
     "msdia140.dll",
+    "R2RDump.exe",
```

Comment:
This is not native binary, might be worth doing collection on this?

Reply:
I was going on the libraries.crossgen2.1.windows.x64.checked.log file from the last collection, where r2rdump.exe and xunit.console.exe failed:
```
[104:331]: C:\h\w\AE830971\p\superpmi\corerun.exe C:\h\w\AE830971\p\superpmi\crossgen2\crossgen2.dll @C:\h\w\AE830971\t\tmp_o4wsgj6\crossgen2_C__h_w_AE830971_w_AB3A0974_u_binaries_R2RDump_R2RDump_exe_arusu6rw.rsp
'C:\h\w\AE830971\p\superpmi\corerun.exe C:\h\w\AE830971\p\superpmi\crossgen2\crossgen2.dll @C:\h\w\AE830971\t\tmp_o4wsgj6\crossgen2_C__h_w_AE830971_w_AB3A0974_u_binaries_R2RDump_R2RDump_exe_arusu6rw.rsp': Error return code: 2
============== Contents of C:\h\w\AE830971\t\tmp_o4wsgj6\crossgen2_C__h_w_AE830971_w_AB3A0974_u_binaries_R2RDump_R2RDump_exe_9h6l6pcd.stdout
No input files are loadable
============== End contents of C:\h\w\AE830971\t\tmp_o4wsgj6\crossgen2_C__h_w_AE830971_w_AB3A0974_u_binaries_R2RDump_R2RDump_exe_9h6l6pcd.stdout
```

also:
```
[324:331]: C:\h\w\AE830971\p\superpmi\corerun.exe C:\h\w\AE830971\p\superpmi\crossgen2\crossgen2.dll @C:\h\w\AE830971\t\tmp_o4wsgj6\crossgen2_C__h_w_AE830971_w_AB3A0974_u_binaries_xunit_core_dll_4bz1xglv.rsp
'C:\h\w\AE830971\p\superpmi\corerun.exe C:\h\w\AE830971\p\superpmi\crossgen2\crossgen2.dll @C:\h\w\AE830971\t\tmp_o4wsgj6\crossgen2_C__h_w_AE830971_w_AB3A0974_u_binaries_xunit_console_exe_e8pdkc1s.rsp': Error return code: 2
============== Contents of C:\h\w\AE830971\t\tmp_o4wsgj6\crossgen2_C__h_w_AE830971_w_AB3A0974_u_binaries_xunit_console_exe_w1nzc9yy.stdout
No input files are loadable
============== End contents of C:\h\w\AE830971\t\tmp_o4wsgj6\crossgen2_C__h_w_AE830971_w_AB3A0974_u_binaries_xunit_console_exe_w1nzc9yy.stdout
```

Reply:
So here is the error we get in crossgen2 for `R2RDump.exe` and `R2RTest.exe`:

```
The format of a DLL or executable being loaded is invalid
```

And here is the error for `xunit.console.dll`:

```
The underlying compression routine could not be loaded correctly.
```

While these might be expected, I just want to double check with @davidwrighton . For this PR, I think you can go ahead and merge it.